src/index.html

```diff
@@ -18,5 +22,85 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="/"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Company Logo"
+        />
+      </a>
+      <nav>
+        <ul class="nav_list">
+          <li class="nav_item">
```

Comment:
The class name `nav_item` should be `nav__item` to follow BEM naming conventions.